src/client.js

```diff
@@ -646,6 +646,12 @@ class Client {
         if (instanceKey.startsWith("http://")) instanceKey = instanceKey.substr(7);
         if (instanceKey.startsWith("https://")) instanceKey = instanceKey.substr(8);
         const rootKey = `acc.js.sdk.${sdk.getSDKVersion().version}.${instanceKey}.cache`;
+        // Clear storage cache if the sdk versions or the instances are different
+        for (let key in this._storage) {
```

Comment:
_storage may be undefined, for instance when running from node.js. It looks like in this case for ... in will just do nothing, but I feel it would be better to actually test _storage to be truthy.
Make sur there's a test for that too (both cases with _storage define and _storage undefined)

Reply:
done